src/recordlinker/schemas/pii.py

```diff
@@ -200,6 +288,28 @@ def field_iter(self, feature: Feature) -> typing.Iterator[str]:
             for name in self.name:
                 if name.family:
                     yield name.family
+        elif feature == Feature.SSN:
+            if self.ssn:
+                yield self.ssn
+        elif feature == Feature.RACE:
+            if self.race:
+                yield str(self.race)
```

Comment:
Are `race` and `gender` not already `str` values?

Reply:
No, they're enums